### ISSUE TITLE
No warning on methods with params and without the params on the XML comment.

### ISSUE THREAD

**Issue by BDisp** [JosefPihrt/Roslynator#867]
**Product and Version Used**: VS2022

**Steps to Reproduce**:

```csharp
/// <summary>
/// Function to do something
/// </summary>
public FnDoSomething(string a, string b)
{
}
```

**Actual Behavior**:
```csharp
No warnings.
```

**Expected Behavior**:

It is expected to receive a warning saying that the "Parameters 'a' and 'b' doesn't exist in the XML comment."

```csharp
/// <summary>
/// Function to do something
/// </summary>
/// <param name="a"></param>
/// <param name="b"></param>
public FnDoSomething(string a, string b)
{
}
```

**Comment by JosefPihrt** [comment]
This is the one you are looking for:
https://github.com/JosefPihrt/Roslynator/blob/master/docs/analyzers/RCS1141.md

it is 'hidden/silent' by default so it is visible only in the context menu and not in the error list. The cursor should be in a documentation comment for a suggestion to show up.

**Comment by BDisp** [comment]
Thanks for your help. They are important like the `<summary>` and must appear in the "`Error List`" as warning by default. Please consider it, because it causes faults in the documentation generator.

**Comment by JosefPihrt** [comment]
Regarding the `summary` element there is an analyzer which has `warning` severity by default:
https://github.com/JosefPihrt/Roslynator/blob/master/docs/analyzers/RCS1139.md

**Comment by BDisp** [comment]
I already saw that there isn't any compiler warnings on methods with parameters, but without none of the XML parameters comments when already there is a summary XML comment. Thanks for your effort.
